macros/get_base_dates.sql

```diff
@@ -1,4 +1,8 @@
 {% macro get_base_dates(start_date=None, end_date=None, n_dateparts=None, datepart=None) %}
+    {{ adapter.dispatch('get_base_dates', packages = dbt_date._get_utils_namespaces()) (start_date, end_date, n_dateparts, datepart) }}
+{% endmacro %}
+
+{% macro default__get_base_dates(start_date=None, end_date=None, n_dateparts=None, datepart=None) %}
```

Comment:
Thanks for adding those params! 
Sorry, one more nitpick: the `default__get_base_dates` and `bigquery__get_base_dates` macros don't need the `None` default parameter values since this get passed in from the dispatch. I'd like to leave those out so we can change the default parameters at the entry point macro without having to change them everywhere else. Thanks!